mdjs-layout/src/mdjs-layout.ts

```diff
@@ -213,6 +214,30 @@ export class MdjsLayout extends HTMLElement {
     this.renderIsNavigationShown();
   }
 
+  private get initialColorScheme(): ColorScheme | null {
+    return this.getAttribute('initial-color-scheme') as ColorScheme | null;
+  }
+
+  private set initialColorScheme(value: ColorScheme | null) {
+    if (value === null) {
```

Comment:
maybe allow all falsy values, change the `value` type and simple if condition `if (!value)`?

Reply:
btw, if this is how Lit does it for pros by default, I'm fiune

Reply:
Hmm yeah I'm kinda just respecting attribute types which is `string | null`, in our case actually only 2 strings allowed + null, so false or undefined would actually be incorrect from a types perspective.